ipv8/keyvault/crypto.py

```diff
@@ -132,3 +136,44 @@ def is_valid_signature(self, ec, data, signature):
             return ec.verify(signature, data)
         except:
             return False
+
+    def is_valid_double_signature(self, data, signature):
+        """
+        Returns True if signature is valid
```

Comment:
Minor detail: please be consistent with ending docstrings with a period or not :)